_posts/2020-04-01-Robust Apps for the cloud.md

```diff
@@ -0,0 +1,253 @@
+To view traces, in the Performance pane, select Take Actions, and then select the Profiler Traces button. 
+
+    ![ai-3]({{site.baseurl}}/media/2020/04/aiprofiler-3.jpg)
+
+1. App Insights also allows you to track dependencies in your application. You can leverage this feature to troubleshoot slow requests. To automatically track dependencies from .NET console apps, install the Nuget package `Microsoft.ApplicationInsights.DependencyCollector`, and initialize `DependencyTrackingTelemetryModule` as follows:
+
+    ```c#
+    DependencyTrackingTelemetryModule depModule = new DependencyTrackingTelemetryModule();
+    depModule.Initialize(TelemetryConfiguration.Active);
+    ```
+
+1. Each request event is associated with the dependency calls, exceptions, and other events that are tracked while your app is processing the request. So if some requests are doing badly, you can find out whether it's because of slow responses from a dependency. You can see a waterfall view of the requests in the performance blade as well under the "Dependencies" tab:
+
+    ![ai-4]({{site.baseurl}}/media/2020/04/dependency.jpg)
+
+You can also leverage our newly released [App Insights integration with App Service Diagnostics](https://azure.github.io/AppService/2020/04/21/Announcing-Application-Insights-Integration-with-App-Service-Diagnostics.html), discussed in details here:
+
+**Learn More**
+
+- [Profile production applications in Azure with Application Insights](https://docs.microsoft.com/azure/azure-monitor/app/profiler-overview)
+- [Diagnose exceptions in your web apps with Application Insights](https://docs.microsoft.com/azure/azure-monitor/app/asp-net-exceptions)
+- [Dependency Tracking in Azure Application Insights](https://docs.microsoft.com/azure/azure-monitor/app/asp-net-dependencies#diagnosis)
+
+## Deploy in Multiple Regions
+
+In the event that a catastrophic incident happens in one of the Azure Datacenters, you can still guarantee that your app will run and serve requests by investing in Azure Front Door or Traffic Manager. There are additional benefits to using Front Door or Traffic Manager, such as routing incoming requests based the customers' geography to provide the shortest respond time to customers and distribute the load among your instances.
+
+**Learn More**
+
+- [Controlling Azure App Service traffic with Azure Traffic Manager](https://docs.microsoft.com/en-us/azure/app-service/web-sites-traffic-manager)
+- [Quickstart: Create a Front Door for a highly available global web application](https://docs.microsoft.com/en-us/azure/frontdoor/quickstart-create-front-door)
+
+## Consider using Linux
+
+When creating a new Web App, follow the recommended app type in the create flow. For example, if you're going to deploy a Python or Node.js app, use Linux.
```

Comment:
I agree that we want to encourage users to use Linux. But right now this isn't telling the reader why using Linux makes my app more robust. Maybe we should cut this section?

Reply:
Let me see what I can do here. I've addressed everything else and just pushed a new version.